### DISCUSSION TITLE
Vertical text selection on the Editor

### DISCUSSION THREAD

**Discussion by bigleka** [dbeaver/dbeaver#20930] [Q&A] [answered]
It's more like a copy/paste feature.
On Visual Studio, SQL Server Management Studion, Notepad++ and others, pressing "SHIFT+ALT" + arrow Up or Down and then arrow LEFT or RIGHT we can select the text vertical and event type text or code to replace or add.
It's just a simples thing that help a lot.

**Comment by E1izabeth** [comment, marked as answer]
I believe it's already possible in DBeaver.

1. Press Alt+Shift+A to enter block selection mode
2. Use arrow keys or mouse left-click dragging to extend the selection
3. Edit in column mode
4. Press Alt+Shift+A again to leave block selection mode

Is it what are you looking for?
